sql/src/test/java/io/crate/integrationtests/ColumnPolicyIntegrationTest.java

```diff
@@ -612,4 +612,19 @@ public void testAlterColumnPolicyOnPartitionedTableWithExistingPartitions() thro
         assertThat(String.valueOf(partitionMetaData.getSourceAsMap().get("dynamic")), is(String.valueOf(ColumnPolicy.DYNAMIC.mappingValue())));
     }
 
+    @Test
+    public void testAlterTableAddColumnToStrictObjectKeepsStrictPolicy() throws Exception {
```

Comment:
I initially looked at adding one at the Analyzer level - AlterTableAddColumnAnalyzerTest - but we can't control the initial value of the object column policy (we can't assert the value was _the same_ as before the alter table).